README.md

```diff
@@ -148,37 +148,58 @@ E2E/Integration tests are available using [Cypress](https://www.cypress.io/).
 
 #### Requirements:
 
-These files required to run integration tests.
+To run the integration tests locally, you'll need the following files set up:
 - `frontend/.env.local`
-- `frontend/cypress/.env.e2e`
 - `/etc/hosts`
-
+- `frontend/cypress.env.json`
+  - only required if running with Okta
+  
 ```
-# frontend/cypress/.env.e2e
-REACT_APP_BASE_URL=http://localhost.simplereport.gov
-REACT_APP_BACKEND_URL=http://localhost.simplereport.gov/api
-REACT_APP_OKTA_ENABLED=true
-REACT_APP_OKTA_URL=http://localhost:8088
+# /etc/hosts
+# add the following line if running with Okta
+127.0.0.1 localhost.simplereport.gov
 ```
 
-The `frontend/.env.local` file has a template at `frontend/cypress/.env.e2e.sample`. Reach out to another developer to get proper values for these secrets.
-
 ```
-# /etc/hosts
-# add the following line
-127.0.0.1 localhost.simplereport.gov
+# frontend/cypresss.env.json
+{
+  "OKTA_USERNAME": "",
+  "OKTA_PASSWORD": "",
+  "OKTA_SECRET": ""
+}
 ```
 
+You'll also need address validation through Smarty to be working:
+- `REACT_APP_SMARTY_STREETS_KEY` must be set in `frontend/.env.local`
```

Comment:
I actually don't have this in my .env.local and it works 🤔

Reply:
thanks for checking! I was about to ping you about this haha
Smarty is only really used in the org sign up spec and I know you need the backend credentials for that. let me try to get it to pass without the frontend key

Reply:
looks like it's not needed. without it, suggested addresses don't populate and our zip/state validation won't fire, but neither of these are actually needed for our e2e tests currently